doc/cluster.md

```diff
@@ -5,15 +5,15 @@ def twoEndpointExample ( sc: SparkContext) = {
   val redisConfig1 = new RedisConfig(new RedisEndpoint("127.0.0.1", 6379, "passwd"))
   val redisConfig2 = new RedisConfig(new RedisEndpoint("127.0.0.1", 7379))
   val rddFromEndpoint1 = {
-    //endpoint("127.0.0.1", 6379) as the default connection in this block
+    // endpoint("127.0.0.1", 6379) as the default connection in this block
     implicit val c = redisConfig1
     sc.fromRedisKV("*")
   }
   val rddFromEndpoint2 = {
-    //endpoint("127.0.0.1", 7379) as the default connection in this block
+    // endpoint("127.0.0.1", 7379) as the default connection in this block
     implicit val c = redisConfig2
     sc.fromRedisKV("*")
   }
 }
 ```
-If you want to use multiple redis clusters/instances, an implicit RedisConfig can be used in a code block to specify the target cluster/instance in that block.
+If you want to use multiple Redis clusters/instances, an implicit RedisConfig can be used in a code block to specify the target cluster/instance in that block.
```

Comment:
"in that block" may be redundant here, but I wasn't sure if the author had another meaning for "block" in the second usage.

Reply:
I think it's redundant and can be removed.

Reply:
Sounds good!